en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,231 @@
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.block">
+                            <para>A <tag>fo:block</tag>-element creates a
+                                rectangle area on a page which can contain
+                                lists, tables or graphics.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.blank">
+                            <para>Here you can insert a text like <emphasis>This
+                                    page is intentionally left
+                                blank</emphasis>.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.center">
+                            <para>This is a good place for the chapter
+                                title.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.right">
+                            <para>This is the place for our logo as described at
+                                the top of our solution.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.height">
+                            <para>To avoid conflicts with the body text area, we
+                                limit its height to 1 inch. When you specify
+                                only one parameter like 
+                                    <tag>height</tag> or <tag>width</tag>, your
+                                logo will be scaled with the <link
+                                    xlink:href="#li.fo.external-graphics"
+                                    >correct aspect ratio</link>.</para>
+                        </callout>
+                        <callout
+                            arearefs="co.topic.graphic-pageheader.attribute">
+                            <para>The file name is passed on the <tag
+                                    class="attribute">src</tag> attribute.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.template">
+                            <para>The template which is called to import and
+                                place the logo.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.file">
+                            <para>Our file defined at the top of our
+                                customization layer.</para>
+                        </callout>
+                    </calloutlist>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>In line 8 of our style sheet (
```

Comment:
I would be careful with phrases like "In line X...". Usually, this can be outdated very soon. Better refer to the callout bug (you can use `<xref linkend="..."/>`).